pkg/upgrade/upgrades/migrations.go

```diff
@@ -8,136 +8,136 @@
 // by the Apache License, Version 2.0, included in the file
 // licenses/APL.txt.
 
-// Package migrations contains the implementation of migrations. It is imported
+// Package upgrades contains the implementation of upgrades. It is imported
 // by the server library.
 //
-// This package registers the migrations with the migration package.
-package migrations
+// This package registers the upgrades with the upgrade package.
+package upgrades
 
 import (
 	"context"
 
 	"github.com/cockroachdb/cockroach/pkg/clusterversion"
-	"github.com/cockroachdb/cockroach/pkg/migration"
+	"github.com/cockroachdb/cockroach/pkg/upgrade"
 	"github.com/cockroachdb/errors"
 )
 
-// GetMigration returns the migration corresponding to this version if
+// GetMigration returns the upgrade corresponding to this version if
 // one exists.
-func GetMigration(key clusterversion.ClusterVersion) (migration.Migration, bool) {
+func GetMigration(key clusterversion.ClusterVersion) (upgrade.Migration, bool) {
 	m, ok := registry[key]
 	return m, ok
 }
 
 // NoPrecondition is a PreconditionFunc that doesn't check anything.
-func NoPrecondition(context.Context, clusterversion.ClusterVersion, migration.TenantDeps) error {
+func NoPrecondition(context.Context, clusterversion.ClusterVersion, upgrade.TenantDeps) error {
 	return nil
 }
 
 // registry defines the global mapping between a cluster version and the
-// associated migration. The migration is only executed after a cluster-wide
+// associated upgrade. The upgrade is only executed after a cluster-wide
 // bump of the corresponding version gate.
-var registry = make(map[clusterversion.ClusterVersion]migration.Migration)
+var registry = make(map[clusterversion.ClusterVersion]upgrade.Migration)
 
-var migrations = []migration.Migration{
-	migration.NewTenantMigration(
+var migrations = []upgrade.Migration{
+	upgrade.NewTenantMigration(
 		"ensure that draining names are no longer in use",
 		toCV(clusterversion.DrainingNamesMigration),
 		NoPrecondition,
 		ensureNoDrainingNames,
 	),
-	migration.NewTenantMigration(
+	upgrade.NewTenantMigration(
 		"add column avgSize to table system.table_statistics",
 		toCV(clusterversion.AlterSystemTableStatisticsAddAvgSizeCol),
 		NoPrecondition,
 		alterSystemTableStatisticsAddAvgSize,
 	),
-	migration.NewTenantMigration(
+	upgrade.NewTenantMigration(
```

Comment:
did you want these to name `NewTenantUpgrade` or do you reckon `Migration` still makes sense here? ok either way

Reply:
Done.